### PR TITLE
Add exposed manager refresh method

### PR DESCRIPTION
**Pull request by Ladas**
Add exposed manager refresh method

### CODE REVIEW

File: lib/miq_automation_engine/service_models/miq_ae_service_ems_event.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 module MiqAeMethodService
   class MiqAeServiceEmsEvent < MiqAeServiceEventStream
+    def manager_refresh(sync: false)
```
Comment (gmcculloug) [inline lib/miq_automation_engine/service_models/miq_ae_service_ems_event.rb]: @Fryguy @jrafanie Have we started using Ruby keyword arguments in other places yet?  Is there any objection to this?

Comment (jrafanie) [reply lib/miq_automation_engine/service_models/miq_ae_service_ems_event.rb]: I don't know if we have but I don't see why not.  I know @chrisarcand would approve.  I think kwargs were feature complete by ruby 2.1 with required kwargs.

Comment (Ladas) [reply lib/miq_automation_engine/service_models/miq_ae_service_ems_event.rb]: :heart: kwargs :-) I use them in many places :-)

Comment (chrisarcand) [reply lib/miq_automation_engine/service_models/miq_ae_service_ems_event.rb]: Keyword arguments were added in 2.0 and you're more than welcome to use them. 👍

Comment (gmcculloug) [reply lib/miq_automation_engine/service_models/miq_ae_service_ems_event.rb]: I'm all for it, just wanted to check if there were any reasons we might be avoiding it.  I see now that it is being used by the backend method this service model is calling: https://github.com/ManageIQ/manageiq/blob/9291810e126cd34e5931bbf985be430a7018a422/app/models/ems_event/automate.rb#L5  
👍

**Pull request merged**